de.dashup.model/src/main/java/de/dashup/model/service/DashupService.java

```diff
@@ -79,13 +79,13 @@ public User getSectionsAndPanels(User user) throws SQLException {
         Map<String, Object> whereParameters = new HashMap<>();
         whereParameters.put("user_id", user.getId());
 
-        List<? extends DatabaseObject> result = this.database.getObject(Database.Table.USER_SECTIONS, Section.class, whereParameters);
+        List<? extends DatabaseObject> result = this.database.getObject(Database.Table.USER_SECTIONS, Section.class, whereParameters, "predecessor_id ASC");
```

Comment:
Please rename this to `order` or `index`. I know, that this would cause our Jenkins to fail, but when merging into master, we want to have a clean state.